src/main/java/com/amazon/opendistroforelasticsearch/sql/query/AggregationQueryAction.java

```diff
@@ -86,7 +86,8 @@ public SqlElasticSearchRequestBuilder explain() throws SqlParseException {
                     // raw javascript dsl, so I'd like to scope the changes as of now to one particular fix for
                     // scripted functions
 
-                    if (!(field instanceof MethodField) || field instanceof ScriptMethodField) {
+                    if (!(field instanceof MethodField) || field instanceof ScriptMethodField
+                            || field.getName().equals("script")) {
```

Comment:
Need more context. So what case this change tries to handle?

Reply:
Done, thanks!